composed_api.rst

```diff
@@ -422,6 +538,188 @@ Example
+| source            | *Required*. Your GRiD generated API key.   |
++-------------------+--------------------------------------------+
+| name              | *Optional*. The name for the export.       |
++-------------------+--------------------------------------------+
+| notes             | *Optional*. User notes.                    |
++-------------------+--------------------------------------------+
+
+Response Format
+^^^^^^^^^^^^^^^
+
+On success, the HTTP status code in the header response is ``200`` OK
+and the response body contains an `Export Detail
+object <#export-detail-object>`_ in JSON format.
+
+Example
+^^^^^^^
+
+::
+
+    curl -u <username> http://gridte.rsgis.erdc.dren.mil/te_ba/api/v2/export/edit/1335/?source=grid&name=new name&notes=notes
+
+::
+
+    {
+        "export": 
+        {
+          "status": "SUCCESS",
+          "pcl_terrain": "",
+          "dim_classification": true,
+          "file_export_options": "individual",
+          "name": "new name",
+          "classification": "",
+          "datatype": "LAS 1.2",
+          "notes": "notes",
+          "rgb": false,
+          "hsrs": "32641",
+          "url": "http://localhost:8000/export/download/1335/",
+          "intensity": true,
+          "pk": 1335,
+          "generate_dem": false,
+          "started_at": "2016-05-16T16:18:12.752305",
+          "sri_hres": null
+        },
+        "exportfiles": [
+        {
+          "url": "http://gridte.rsgis.erdc.dren.mil/te_ba/export/download/file/30359/",
+          "pk": 30359,
+          "name": "ExportedFile.laz"
+        }
+        ],
+        "tda_set": [],
+        "API Version": "v2",
+        "success": true
+    }
+
+Delete Export
+~~~~~~~~~~~~~
+
+Delete an existing Export.
+
+Endpoint
+^^^^^^^^
+
+::
+
+    GET <instance_url>/<instance_root>_ba/api/v2/export/delete/<pk>
+
+Request Parameters
+^^^^^^^^^^^^^^^^^^
+
++------------------+----------------------------------+
+| Path parameter   | Value                            |
++==================+==================================+
+| pk               | The primary key of the export.   |
++------------------+----------------------------------+
+
++-------------------+--------------------------------------------+
+| Query parameter   | Value                                      |
++===================+============================================+
+| source            | *Required*. Your GRiD generated API key.   |
++-------------------+--------------------------------------------+
+
+Response Format
+^^^^^^^^^^^^^^^
+
+On success, the HTTP status code in the header response is ``200`` OK
+and the response body contains a the status in JSON format.
+
+Example
+^^^^^^^
+
+::
+
+    curl -u <username> http://gridte.rsgis.erdc.dren.mil/te_ba/api/v2/export/delete/1335/?source=grid
+
+::
+
+    {
+        "API Version": "v2",
+        "success": true
+    }
+
+Get Product Details
+~~~~~~~~~~~~~~~~~~~
+
+Get information for a single Product.
```

Comment:
A product is similar to a collect in this case (a collect is made up of 1 or more products).  Products available to be exported are returned in both AOI details and AOI list queries.  However, if you are querying an AOI, we do not return the geometry of the product(s) as this could potentially be a very large return.  If you want to see a product geometry, you need to query the product details.

This is not referring to user generated products, such as TDAs.